### ISSUE TITLE
set default sex to female

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/digital-growth-charts-react-client#70]
In order not to perpetuate gender bias cf [Invisible Women - Exposing data bias in a world designed for men, Caroline Criado Perez](https://carolinecriadoperez.com/book/invisible-women/), the charts have sought de-emphasis the old pink/blue. In addition the default selection could also be set to female.

**Comment by statist7** [comment]
I agree that default male is wrong, but default female is not much better. Perhaps no default at all, or even use a random sex while flagging it very obviously.

**Comment by dc2007git** [comment]
Having no default seems like the best way to resolve this - then giving the user the option to choose between male and female in a dropdown

**Comment by eatyourpeas** [comment]
I think this will run and run. Closing this for now.
